pkg/events/cloudevent.go

```diff
@@ -0,0 +1,94 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package events
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+)
+
+const (
+	// CloudEventsVersion defines version of CloudEvent specification used in Dispatch
+	CloudEventsVersion = "0.1"
+)
+
+// CloudEvent structure implements CloudEvent spec:
+// https://github.com/cloudevents/spec/blob/b0124528486d3f6b9a247cadd68d91b44b3d3ef4/spec.md
+type CloudEvent struct {
```

Comment:
We should blog about our inclusion of cloud events.

Reply:
Definitely!